beeline/src/java/org/apache/hive/beeline/BeeLine.java

```diff
@@ -880,6 +881,12 @@ private boolean connectUsingArgs(BeelineParser beelineParser, CommandLine cl) {
       setHiveConfVar(key, hiveConfs.getProperty(key));
     }
 
+    String opLogLevelVarName = HiveConf.ConfVars.HIVE_SERVER2_LOGGING_OPERATION_LEVEL.varname;
```

Comment:
i think that should be downstream only change where could disable `HIVE_SERVER2_LOGGING_OPERATION_LEVEL`

Reply:
This is related to the previous comment.